docs/releasing.md

```diff
@@ -25,35 +43,108 @@ Release manager follows the below _Release Process_.
 
 ### Versioning
 
-We follow [semver](https://semver.org/) versioning semantics (`vMajor.Minor.Patch`), to align with the greater Node community. Generally, this means our bi-weekly releases will bump a minor. Though we will release a new patch version if high-priority fixes are required before the next schedule release. Additionally, if a schedule release contains no new features, then we'll only bump the patch version.
+We follow [semver](https://semver.org/) versioning semantics (`vMajor.Minor.Patch`). Breaking changes will bump the major version. New features or bug fixes will bump the minor version. If a release contains no new features, then we'll only bump the patch version.
 
 
 ## Release Process
 
+Note: You'll wanna be on a Linux machine, since the Lightrider step will require that.
+
+### On the scheduled release date
+
+Before starting, you should announce to the LH eng channel that you are releasing,
+and that no new PRs should be merged until you are done. If you have to go through all these steps
+again to assert that release-blocking fixes work, other PRs may be merged too, but that should
+be coordinate with you.
+
 ```sh
-# Run the tests
+# Run the tests.
 bash ./lighthouse-core/scripts/release/test.sh
+# Change into the pristine folder.
+cd ../lighthouse-pristine
+```
+
+Confirm DevTools integration will work:
+```sh
+# You should have Chromium already checked out at ~/chromium/src
+# See: https://www.chromium.org/developers/how-tos/get-the-code
+
+# Roll to Chromium folder.
+yarn devtools
```

Comment:
@paulirish wasn't there a way (used to be a way?) to live-edit devtools? Could we do that with Lighthouse so that the releaser isn't dependent on rebuilding?

Reply:
yahh there is a thing now. `--custom-devtools-frontend`

https://chromium-review.googlesource.com/c/chromium/src/+/1791150

this might just work for us. but might not because remote modules.